### PR TITLE
feat: support adding people with mentions

### LINKED ISSUE #26: Support adding people with mentions
**Is your feature request related to a problem? Please describe.**
Adding a user using `@all-contributors please add @tbenning for design` results in a fail
screen shot 2019-01-15 at 1 00 33 am

While non-mentions succeed:
screen shot 2019-01-15 at 1 00 58 am

**Describe the solution you'd like**
Adding them for their contributions!

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sinchang**
closes #26

### CODE REVIEW

File: src/utils/parse-comment/index.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const plugin = {
 nlp.plugin(plugin)
 
 function parseAddComment(doc, action) {
-    const who = doc
+    let who = doc
```
Comment (jakebolam) [inline src/utils/parse-comment/index.js]: [minor, not blocking] usually try to keep variables assignment immutable with `const`
see https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1

File: test/utils/parse-comment/index.test.js [hunk 2]
```diff
@@ -55,6 +55,18 @@ describe('parseComment', () => {
         })
     })
 
+    test('Support adding people with mentions', () => {
+        expect(
+            parseComment(
+                `@${testBotName} please add @jakebolam for infrastructure`,
```
Comment (jakebolam) [inline test/utils/parse-comment/index.test.js]: > Please add @sinchang for infrastructure
😉

**Comment by jakebolam** [comment]
@sinchang let me know when good to merge

**Comment by sinchang** [comment]
@jakebolam you can merge now. 😄

File: src/utils/parse-comment/index.js [hunk 3]
```diff
@@ -75,6 +75,8 @@ function parseAddComment(doc, action) {
         })
         .data()[0].text
 
+    const who = whoMatched.startsWith('@') ? whoMatched.substr(1) : whoMatched
```
Comment (jakebolam) [inline src/utils/parse-comment/index.js]: :+1

**Comment by jakebolam** [comment]
Thank you!

**Pull request merged**